package.json

```diff
@@ -7,21 +7,33 @@
   "homepage": "https://algorithmia.com/",
   "license": "The MIT License (MIT)",
   "scripts": {
-    "start": "NODE_ENV=production node ./server/index.js",
+    "apidocs:build": "cd api-docs && bundle exec rake build",
```

Comment:
I had to update this line to 
```
"apidocs:build": "cd api-docs && bundle install && bundle exec rake build",
```
in order to get it to work.

Reply:
Okay, will update!